src/System.Private.CoreLib/src/System/MissingMemberException.CoreCLR.cs

```diff
@@ -0,0 +1,25 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*=============================================================================
+**
+**
+**
+** Purpose: The exception class for versioning problems with DLLS.
+**
+**
+=============================================================================*/
+
+using System.Runtime.Serialization;
+using System.Runtime.CompilerServices;
+
+namespace System
+{
+    public partial class MissingMemberException : MemberAccessException, ISerializable
+    {
+        // Called to format signature
```

Comment:
This comment is useless too. What else would one call `FormatSignature` method than to format signature?